integration-test/491-feature-tests.py

```diff
@@ -0,0 +1,26 @@
+from . import FixtureTest
+
+
+class InclusivePois(FixtureTest):
```

Comment:
@rmarianski @zerebubuth I think this class name is just re-using some other test file's class name (one about International Women's Day and inclusive POIs). Can we rename it to be something more generic here?

Reply:
Yup. The class name is not important for the functioning of the test but, like a misleading comment, might contribute to confusion. Since #491 is very generic, it would probably be okay to rename this to `FeaturesTest` or similar. Finer granularity issues could help inspire finer granularity names.